versus-fe/src/components/Match/innerMatch.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+export default {
+  finished: props => {
+    return (
+      <div className="MatchContainer__Content__result">
+        <span>
+          <b>{props.user1.score}</b> / {props.user2.score}
+        </span>
+        <i> + 30 points! </i>
+      </div>
+    );
+  },
+
+  choices: props => {
+    return (
+      <div>
+        <button
+          className="MatchContainer__Content__accept"
+          onClick={props.innerFunction.accept}
+        >
+          accept
+        </button>
+        <button
+          className="MatchContainer__Content__decline"
+          onClick={props.innerFunction.decline}
+        >
+          decline
+        </button>
+      </div>
+    );
+  },
+
+  waiting: () => {
+    return (
+      <div className="MatchContainer__Content__waiting">
+        <h2>
+          <i>waiting...</i>
+        </h2>
+      </div>
+    );
+  },
+
+  denied: props => {
+    console.log(props);
```

Comment:
Remove console.logs, your devtools console is going to be a mess after some time,.

Reply:
fixed